src/types/index.ts

```diff
@@ -431,6 +439,15 @@ export interface IDexContracts {
   wae: string[];
 }
 
+export type ICommonTransaction = ITransaction | IActiveMultisigTransaction
+
+export type ITransactionsState = {
+  loaded: ITransaction[];
+  nextPageUrl?: string;
+  pending: any; // TODO type that object
```

Comment:
isn't that a boolean?

Reply:
Nope, this is a list of pending transactions.